web-client/src/store/index.ts

```diff
@@ -0,0 +1,18 @@
+import { combineReducers, createStore } from "redux";
+import { composeWithDevTools } from "redux-devtools-extension";
```

Comment:
imports with double quotes instead of single quotes here

Reply:
@comoser I created an [`issue`](https://github.com/reach4help/reach4help/issues/71) to enforce some rules on eslint. Maybe I'll leave these coding-style related comments for that issue.

Reply:
makes sense! go for it